pose-detection/src/blazepose/calculators/calculate_landmark_projection.ts

```diff
@@ -33,6 +33,7 @@ export function calculateLandmarkProjection(
     config: {ignoreRotation: boolean} = {
       ignoreRotation: false
     }) {
+  const outputLandmarks = [];
   for (let i = 0; i < landmarks.length; ++i) {
     const landmark = landmarks[i];
```

Comment:
Does look like it creates copy of the landmark. You are still editing original one, no?